packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -887,7 +897,8 @@ export class Repository extends Construct implements IRepository {
   private configureRepositoryInstallerScript(
     installerGroup: AutoScalingGroup,
     installPath: string,
-    version: IVersion) {
+    version: IVersion,
+    settings?: Asset) {
```

Comment:
The closing parenthasis shoud be on a new line and the `settings` line should have a comma dangle.

Reply:
Fixed

Reply:
Fixed